fedgov-cv-lib-asn1c/src/main/java/wrapper.c

```diff
@@ -162,57 +162,56 @@ JNIEXPORT jbyteArray JNICALL Java_gov_usdot_cv_mapencoder_Encoder_encodeMap(JNIE
 			jmethodID isFullRdAuthIDExists = (*env)->GetMethodID(env, intersectionClass, "isFullRdAuthIDExists", "()Z");
 			jboolean fullRdAuthIDExists = (*env)->CallBooleanMethod(env, intersectionObj, isFullRdAuthIDExists);
 
-
 			// Check if relative RAID exists
 			jmethodID isRelRdAuthIDExists = (*env)->GetMethodID(env, intersectionClass, "isRelRdAuthIDExists", "()Z");
 			jboolean relRdAuthIDExists = (*env)->CallBooleanMethod(env, intersectionObj, isRelRdAuthIDExists);
 			
 			if (fullRdAuthIDExists)
 			{
 				// Get full RAID
-				jmethodID getFullRdAuthID = (*env)->GetMethodID(env, intersectionClass, "getFullRdAuthID", "()Ljava/lang/String;");
-				jstring fullRdAuthID = (*env)->CallObjectMethod(env, intersectionObj, getFullRdAuthID);
-				const char *fullRdAuthIDStr = (*env)->GetStringUTFChars(env, fullRdAuthID, 0);
+				jmethodID getFullRdAuthID = (*env)->GetMethodID(env, intersectionClass, "getFullRdAuthID", "()[I");
```

Comment:
What is the purpose of the string "()[I"

Reply:
GetMethodID JNI Function requires a type signature: https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#GetMethodID

"()[I" denotes a array of integers: https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/types.html